db/schema.rb

```diff
@@ -22,6 +22,7 @@
     t.integer "quantity"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "status"
```

Comment:
I wonder how these migrations work, I guess it just overwrites and then we have to re-merge anyways?